### ISSUE TITLE
Review author and contributors fields

### ISSUE THREAD

**Issue by arkaitzgarro** [WeTransfer/concorde.js#398]
As per [npm documentation](https://docs.npmjs.com/files/package.json#people-fields-author-contributors):

> The “author” is one person. “contributors” is an array of people.

We are specifying `"author": "WeTransfer folks",`, which is not correct. We also may want to add ourselves as contributors, or add a generic dev account, like `"contributors": ["WeTransfer folks <devs@wetransfer.com> (https://github.com/WeTransfer)"]`

**Comment by seelts** [comment]
Yep, as [I've mentioned in the clipboard PR](https://github.com/WeTransfer/concorde.js/pull/390#discussion_r317065154), I think we should not use `author` field for "WeTransfer folks".
Let's use `contributors` field for that.

As for `author` field and putting a real person's name there, I have nothing against it.
You can see other companies doing that.
For me it looks correct from the ethical point of view.
That for sure should be aligned with WeTransfer's policy regarding this.
And I will not be surprised if this is prohibited.

So, let's just decide if we want it and if we are allowed to do that.
